### PR TITLE
Updated 'predix seed' links and texts to 'predix web app starter'

### PR DESCRIPTION
**Pull request by socksrust**

### CODE REVIEW

File: pages/guides/use-seed.html [hunk 1]
```diff
@@ -33,8 +33,8 @@ <h2 class="selected">Start a new app with the Predix UI Seed</h2>
         <li class="u-mv--">A Node/Express web server ready to connect to Predix services and APIs</li>
       </ul>
 
-      <a class="info-box" target="_blank" href="https://github.com/predixdev/predix-seed/">
-        <h3 class="actionable"><strong>Download or fork the Predix UI Seed on Github <i class="fa fa-external-link u-ml--"></i></strong></h3>
+      <a class="info-box" target="_blank" href="https://github.com/predixdev/predix-webapp-starter">
+        <h3 class="actionable"><strong>Download or fork the Predix UI web app starter on Github <i class="fa fa-external-link u-ml--"></i></strong></h3>
         <p>The predix-seed Github repository contains the full instructions to help you download the seed, install necessary dependencies, and develop and application. Take a look at the README.</p>
```
Comment (talimarcus) [inline pages/guides/use-seed.html]: Missing two more to update here:
- "The predix-seed Github repository..."
- "... to help you download the seed, ..."

And may as well fix "and develop and application" to "and develop *an* application" while you're in here :)

Comment (talimarcus) [reply pages/guides/use-seed.html]: Can you also capitalize them please? --> "Predix UI Web App Starter" 

(the first one where it just says "a quick-start web app" is fine)

Comment (jaburcodes) [inline pages/guides/use-seed.html]: sure!! shoud i do a push --force to my fork and do a new PR with all changes in one commit, or may i add one more commit with your requested changes?

Comment (socksrust) [reply pages/guides/use-seed.html]: done! :)

**Review by talimarcus — APPROVED** [review]
Looks great, thank you!

**Pull request merged**